lib/pwnlib/context.rb

```diff
@@ -174,17 +241,35 @@ def log_level=(value)
         @attrs[:log_level] = log_level
       end
 
+      # Set the operating system of the target machine.
+      #
+      # @param [String, Symbol] os
+      #   The name of the os.
+      #
+      # @example
+      #   context.os = :windows
       def os=(os)
-        os = os.downcase
+        os = os.to_s.downcase
         raise ArgumentError, "os must be one of #{OSES.sort.inspect}" unless OSES.include?(os)
         @attrs[:os] = os
       end
 
+      # Set the signedness for packing opreation.
+      #
+      # @param [String, Symbol, true, false] value
+      #   The signedness.
```

Comment:
reference the constant `SIGNEDNESSES` here to let users know what are valid arguments